DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/traffic/ixnetwork/ixnetwork_ixia_client_impl.py

```diff
@@ -199,39 +243,23 @@ def set_l4_traffic(self, config_element, ipv4_stack, pkt_data):
         )
         l4_stack = config_element.Stack.read(ipv4_stack.AppendProtocol(ipproto_template))
         if "dstPort" in pkt_data:
-            dst_port = l4_stack.Field.find(
-                FieldTypeId="{}.header.dstPort".format(pkt_data["ipproto"])
-            )
-            dst_port.Auto = False
             if ":" in pkt_data["dstPort"]:
```

Comment:
@muchetan As I understand we need this exception in config processing since there are some tests that are using this. Is this possible to update them to use a single way to configure incremental data?

Reply:
yes we can change the callers if we decide to go this way.